src/main/java/uk/gov/pay/ledger/transaction/model/CsvTransactionFactory.java

```diff
@@ -154,7 +155,12 @@ public Map<String, Object> toMap(TransactionEntity transactionEntity) {
                         getPaymentTransactionAttributes(transactionEntity, transactionDetails.get("payment_details"))
                 );
                 result.put(FIELD_GOVUK_PAYMENT_ID, transactionEntity.getParentExternalId());
-                result.put(FIELD_AMOUNT, penceToCurrency(transactionEntity.getAmount() * -1));
+                boolean isWonDispute = transactionEntity.getState() == WON;
```

Comment:
You can just do:
`if(transactionEntity.getState() == WON)`.  The variable `isWonDispute` is used only for the if statement.

Reply:
Ah good point, thanks!

Reply:
Amended, thanks!